alphastats/DataSet_Statistics.py

```diff
@@ -194,36 +130,36 @@ def ancova(
         ancova_df = pingouin.ancova(df, dv=protein_id, covar=covar, between=between)
         return ancova_df
 
-    @ignore_warning(RuntimeWarning)
-    def multicova_analysis(
-        self,
-        covariates: list,
-        n_permutations: int = 3,
-        fdr: float = 0.05,
-        s0: float = 0.05,
-        subset: dict = None,
-    ) -> Union[pd.DataFrame, list]:
-        """Perform Multicovariat Analysis
-        will return a pandas DataFrame with the results and a list of volcano plots (for each covariat)
-
-        Args:
-            covariates (list): list of covariates, column names in metadata
-            n_permutations (int, optional): number of permutations. Defaults to 3.
-            fdr (float, optional): False Discovery Rate. Defaults to 0.05.
-            s0 (float, optional): . Defaults to 0.05.
-            subset (dict, optional): for categorical covariates . Defaults to None.
-
-        Returns:
-            pd.DataFrame: Multicova Analysis results
-        """
-
-        res, plot_list = MultiCovaAnalysis(
-            dataset=self,
-            covariates=covariates,
-            n_permutations=n_permutations,
-            fdr=fdr,
-            s0=s0,
-            subset=subset,
-            plot=True,
-        ).calculate()
-        return res, plot_list
+    # @ignore_warning(RuntimeWarning)
+    # def multicova_analysis(  # TODO never used outside of tests .. how does this relate to multicova.py?
```

Comment:
In MultiCovaAnalysis multicova.py is imported from the calculate function.
```
    def calculate(self):
        from alphastats.multicova import multicova
```

This is defenitely a feature we want in the tool, so it should remain active but we need a ticket for putting it in the interface.